_posts/2022-10-18-cockpit-278.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Cockpit 278
+author: jelle
+date: '2022-10-19'
+tags: cockpit, podman
+slug: cockpit-278
+category: release
+summary: 'Individual disk/read write usage metrics, '
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 278 and cockpit-podman 55:
+
+
+## Metrics: Display individual disk read/write usage
+
+Metrics page now supports monitoring throughput of each individual disk when multiple disks are detected.
+![screenshot of display individual disk read/write usage]({{ site.baseurl }}/images/278-metrics-display-individual-disk-read-write-usage.png)
+
+
+## Pod CPU, memory, port and volume details
+
+Podman now shows the CPU and memory usage of all the containers in a pod, the CPU usage shown is the highest CPU usage of one of the containers in the pod. Next to the CPU and memory information the exposed ports and volumes are shown, clicking on the port icon shows the port mapping of the pod.
+
+![screenshot of ## pod cpu, memory, port and volume details]({{ site.baseurl }}/images/278-pod-cpu-memory-port-and-volume-details.png)
+
```

Comment:
```suggestion
Clicking on the port icon shows the port mapping of the pod.
```

Moving this part be more contextual with the screenshot.

Hopefully GitHub does the right thing with regard to spacing in the file. If not, we can easily fix it.